cda-core/src/pt/webdetails/cda/dataaccess/kettle/SQLKettleAdapter.java

```diff
@@ -171,6 +174,9 @@ protected DatabaseMeta getDatabaseMeta() throws KettleAdapterException {
         }
       }
     }
+    if ( StringUtils.isEmpty( databaseMeta.getDatabaseName() ) ) {
```

Comment:
Add a comment here explaining why this is here and linking to the PDI jira case.